package.json

```diff
@@ -103,11 +103,11 @@
     "make:dev": "make dev",
     "generate:machine-api": "npx openapi-typescript ./openapi/machine-api.json -o src/lib/machine-api.d.ts",
     "tron:start": "electron-forge start",
-    "tron:package": "electron-forge package",
```

Comment:
Removing this in favor of a more widespead use of electron-builder for packaged apps. Trying to make it easier for people to run 'the prime packaging' path that we actually deploy.

Reply:
So we'll whittle it down to just `tron:start` to use electron-forge?